scale-ui/app/modules/jobs/controllers/jobsController.js

```diff
@@ -79,7 +79,13 @@
             {
                 field: 'status',
                 width: 150,
-                cellTemplate: '<div class="ui-grid-cell-contents"><div class="pull-right"><button ng-show="((!grid.appScope.vm.readonly) && (row.entity.status === \'FAILED\' || row.entity.status === \'CANCELED\'))" ng-click="grid.appScope.vm.requeueJobs({ job_ids: [row.entity.id] })" class="btn btn-xs btn-default" title="Requeue Job"><i class="fa fa-repeat"></i></button> <button ng-show="!grid.appScope.vm.readonly && row.entity.status !== \'COMPLETED\' && row.entity.status !== \'CANCELED\'" ng-click="grid.appScope.vm.cancelJob(row.entity)" class="btn btn-xs btn-default" title="Cancel Job"><i class="fa fa-ban"></i></button></div> {{ row.entity.status }}</div>',
+                cellTemplate: '<div class="ui-grid-cell-contents">' +
+                    '<div class="pull-right">' +
+                        '<button ng-show="((!grid.appScope.vm.readonly && !row.entity.is_superseded) && (row.entity.status === \'FAILED\' || row.entity.status === \'CANCELED\'))" ng-click="grid.appScope.vm.requeueJobs({ job_ids: [row.entity.id] })" class="btn btn-xs btn-default" title="Requeue Job">' +
+                            '<i class="fa fa-repeat"></i></button> ' +
+                        '<button ng-show="!grid.appScope.vm.readonly && !row.entity.is_superseded && row.entity.status !== \'COMPLETED\' && row.entity.status !== \'CANCELED\'" ng-click="grid.appScope.vm.cancelJob(row.entity)" class="btn btn-xs btn-default" title="Cancel Job">' +
+                            '<i class="fa fa-ban"></i></button>' +
+                    '</div> {{ row.entity.status }} <span ng-if="row.entity.is_superseded" class="label label-info">SUPERSEDED</span> </div>',
```

Comment:
Yeah, I think putting it where the action buttons are makes sense.

Instead of using S, we can use an icon. I had submitted an issue a while back about job status icons and had one picked out for SUPERSEDED as well. Give f24d a try and we'll see how it looks. For a tooltip you can use the same "This job is obsolete because its recipe has been reprocessed." text.